docs/windows.md

```diff
@@ -0,0 +1,191 @@
+# Windows Support
+
+Starting with the release version [0.10.0](https://github.com/cloud-hypervisor/cloud-hypervisor/releases/tag/v0.10.0), Cloud Hypervisor has full Windows guest support.
+
+Requirements
+
+- Host with KVM enabled 
+- [UEFI](uefi.md) capable guest
+- Guest with VirtIO integrated drivers
+
+At the current stage, only UEFI capable Windows images are supported. This implies the presence of the OVMF firmware during the Windows installation and in any subsequent usage. BIOS boot is not supported.
+
+The subsequent sections will tell, in detail, how to prepare an appropriate Windows image.
+
+## Image Preparation
+
+### Installation using the stock Windows ISO
+
+Prerequisites
+
+- QEMU
+- Windows installation ISO
```

Comment:
I googled this https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2019 . Another possibility is a Visual Studio subscription. It can be put here, I just didn't see it as relevant.

Reply:
Which versions of Windows work? I have only tested Server 2019.

Reply:
I've added a word about this. On my side it is 2019, 2004 and partially 1909.

Thanks.